src/components/Dashboard/TodoList.vue

```diff
@@ -979,22 +994,21 @@ export default {
         case FilingTypes.ANNUAL_REPORT:
           // resume this Annual Report
           this.setARFilingYear(task.ARFilingYear)
-          this.setArMinDate(task.arMinDate)
-          this.setArMaxDate(task.arMaxDate)
+          this.setArMinDate(task.arMinDate) // COOP only
+          this.setArMaxDate(task.arMaxDate) // COOP only
+          this.setNextARDate(task.nextArDate) // BCOMP only
```

Comment:
This is actually not used (yet) because BCOMP ARs cannot be saved as drafts, and therefore cannot be resumed. But now it's ready in case BCOMP ARs do get saved/resumed :)